### PR TITLE
Refactor zmetrics to include appStart, appVersion and appRevision metrics and more

### PR DESCRIPTION
**Pull request by lucaslopezf**
:link: [zboto Link](https://app.clickup.com/t/8694072fv)

### CODE REVIEW

**Review by emmanuelm41 — COMMENTED** [review]
Let's fix ci first

File: pkg/metrics/register.go [hunk 1]
```diff
@@ -24,7 +25,7 @@ var (
 )
 
 func (t *taskMetrics) RegisterMetric(name string, help string, labels []string, handler MetricHandler) error {
-	labels = append(labels, appNameLabel, appVersionLabel)
+	labels = append(labels, appNameLabel, appVersionLabel, appRevisionLabel)
```
Comment (emmanuelm41) [inline pkg/metrics/register.go:28]: appName can be a label, but appVersion and appRevison, as well as appStartUp should be a metric itself

**Review by emmanuelm41 — CHANGES_REQUESTED** [review]
Left an important comment!

File: pkg/zrouter/zrouter.go [hunk 2]
```diff
@@ -13,9 +14,11 @@ import (
 )
 
 const (
-	defaultAddress = ":8080"
-	defaultTimeOut = 240000
-	appStartMetric = "app_start"
+	defaultAddress    = ":8080"
+	defaultTimeOut    = 240000
+	appStartMetric    = "app_start"
```
Comment (emmanuelm41) [inline pkg/zrouter/zrouter.go:19]: `uptime` would be better name. What do you think?

**Review by emmanuelm41 — APPROVED** [review]

File: pkg/metrics/register.go [hunk 3]
```diff
@@ -9,9 +9,7 @@ import (
 )
 
 const (
-	appNameLabel     = "app_name"
-	appVersionLabel  = "app_version"
-	appRevisionLabel = "app_revision"
+	APPNameLabel = "app_name"
```
[the remark this answers is not in the document]
Comment (lucaslopezf) [reply pkg/metrics/register.go:12]: No, we need the constant to be public, not private

[the remark this answers is not in the document]
Comment (emmanuelm41) [reply pkg/metrics/register.go:12]: But it could be AppNameLabel, instead of APPNameLabel!

**Pull request merged**